src/style.css

```diff
@@ -1,3 +1,112 @@
-body {
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+* {
   margin: 0;
```

Comment:
it's allowed to use only box-sizing: border-box; for * selector. Reset default margin for those tags you need
```suggestion
```

Reply:
use ```margin: 0``` for tags directly: body, p, h1-6 and so on